### PR TITLE
Verify Gradle 6.4.1 support

### PR DESCRIPTION
**Pull request by MyDogTom**
* Bump wrapper to Gradle 6.4.1 version by executing `./gradlew wrapper  --gradle-version=6.4.1`
* Include 6.4.1 version into DefaultConfigurationSpec test

### CODE REVIEW

**Comment by MyDogTom** [comment]
I'm still struggling with executing tests locally :( Test that use Docker containers fail with
```
Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
    UnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause NoSuchFileException (/var/run/docker.sock)
    UnixSocketClientProviderStrategy: failed with exception InvalidConfigurationException (ping failed). Root cause NoSuchFileException (/var/run/docker.sock)
As no valid configuration was found, execution cannot continue
Error during test engine run
```

I'll investigate further and afterwards debug `InfluxDbPublisherTest` to understand why it's failing in the PR.

File: talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/DefaultBuildDataProviderTest.kt [hunk 1]
```diff
@@ -36,9 +36,9 @@ class DefaultBuildMetricsProviderTest : BehaviorSpec({
                     "buildId" to "12",
                     "buildInvocationId" to "123",
                     "osVersion" to "Linux 1.4",
-                    "maxWorkers" to 2,
+                    "maxWorkers" to 2.toInt(),
                     "javaRuntime" to "1.2",
-                    "cpuCount" to 4,
+                    "cpuCount" to 4.toInt(),
```
Comment (MyDogTom) [inline talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/DefaultBuildDataProviderTest.kt:41]: but I don't know why it's required now :(

Comment (cdsap) [reply talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/DefaultBuildDataProviderTest.kt:41]: Checking too

Comment (cdsap) [reply talaiot/src/test/kotlin/com/cdsap/talaiot/metrics/DefaultBuildDataProviderTest.kt:41]: tested even changing the order like:
```
 mapOf(
                "maxWorkers" to 2,
                "cpuCount" to 4,
                 ...

```
And getting the same error.  I tried with Kotlin 1.3.72 and  same result. I would like to know how is possible the cast for the map is not working as expected(and was working before).

Merging the PR, is not blocker.

**Pull request merged**